File: lib/layer2_core/light/e_outline_light.dart
import '../../layer1_raw/design_token.dart';
import '../../layer1_raw/light/system.dart';
import '../../layer1_raw/light/standard.dart';
import '../../layer1_raw/light/blue.dart';
import '../../layer1_raw/light/green.dart';
import '../../layer1_raw/light/red.dart';
import '../../layer1_raw/light/nothing.dart';

class EOutlineLight {
  static DesignToken getToken(dynamic activeTheme) {
    switch (activeTheme.toString().split('.').last) {
      case 'system': return csyl45;
      case 'standard': return cstl45;
      case 'blue': return cbl45;
      case 'green': return cgl45;
      case 'red': return crl45;
      case 'nothing': return cnol45;
      default: return cstl45;
    }
  }
}

File: android/src/main/kotlin/com/example/design_engine/layer2_core/light/EOutlineLight.kt
package com.example.design_engine.layer2_core.light
import com.example.design_engine.R
import com.example.design_engine.layer3_logic.AppTheme
object EOutlineLight {
    fun getResId(activeTheme: AppTheme): Int {
        return when (activeTheme) {
            AppTheme.SYSTEM -> R.string.csyl45
            AppTheme.STANDARD -> R.color.cstl45
            AppTheme.BLUE -> R.color.cbl45
            AppTheme.GREEN -> R.color.cgl45
            AppTheme.RED -> R.color.crl45
            AppTheme.NOTHING -> R.color.cnol45
        }
    }
}
